k8s/contiv-vpp/README.md

```diff
@@ -21,6 +21,13 @@ helm template --name my-release ../contiv-vpp > manifest.yaml
 kubectl apply -f manifest.yaml
 ```
 
+To install without tiller for arm64, you can generate the manifest from this chart and install via kubectl:
+
+```console
+helm template --name my-release ../contiv-vpp -f ./values-arm64.yaml > manifest-arm64.yaml
```

Comment:
shouldn't this contain both: `-f ./values-arm64.yaml,./values.yaml` ?

Reply:
Yes, it should be added. 
Updated.